packages/form/package.json

```diff
@@ -57,7 +57,7 @@
     "babel-loader": "^8.2.5",
     "jsdom": "^20.0.0",
     "postcss": "^8.4.14",
-    "ts-standard": "^11.0.0",
+    "ts-standard": "latest",
```

Comment:
Please use same version as core package.
When ts-standard 13 is released we will have different versions installed that could cause problems.